### PR TITLE
Adding r-hutils package

### PR DESCRIPTION
**Pull request by efendyk**
@conda-forge/help-r, first-timer here, please kindly review

Checklist
- [v] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [v] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [v] Source is from official source.
- [v] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [v] If static libraries are linked in, the license of the static library is packaged.
- [v] Build number is 0.
- [v] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [v] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [v] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by efendyk** [comment]
@conda-forge/help-r
- The recipe could do with some maintainers listed in the extra/recipe-maintainers section.
Me: I've listed myself as the maintainer. What else is missing?
- Please move the recipe out of the example dir and into its own dir
Me: I think I've done that. What was wrong?
- Please do not delete the example recipe found in recipes/example/meta.yaml.
Me: I'm not sure if any was deleted or what was deleted. I don't think any was deleted.
- License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.
Me: Not sure what this means. I used conda-r-skeleton-helper and followed its instructions as well as from this blog post: https://taylorreiter.github.io/2020-01-16-Building-a-conda-forge-package-from-an-R-CRAN-package/

**Comment by efendyk** [comment]
Hmmm I think I know what went wrong.
Shall I close this PR and re-try creating a PR separately again?

File: recipes/example/r-hutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://github.com/hughparsonage/hutils, https://hughparsonage.github.io/hutils/
+
+  license: GPL-3
```
Comment (dbast) [inline recipes/example/r-hutils/meta.yaml:61]:
```suggestion
  license: GPL-3.0-only

```

**Comment by dbast** [comment]
See the suggestion and please also remove all comments except the cran metadata at the end of the meta.yaml.

**Pull request closed without merging**